static/wp-admin/admin-ajax.css

```diff
@@ -1,1606 +1,1482 @@
+
+/* layout two */
+
+.sidebar_layout_two section{  background:#ffffff; }
+.sidebar_layout_two .tagcloud a{border:1px solid #edf0f7; font-family:Roboto; }
+.sidebar_layout_two .tagcloud a:hover { background:#efa537; border-color:#efa537; color:#ffffff; }
+
+/* layout three */
+
+
+.sidebar_layout_three .tagcloud a{border:1px solid #edf0f7; font-family:Roboto;  color:#6d7783; background:#ffffff;}
+.sidebar_layout_three .tagcloud a:hover { background:#efa537; border-color:#efa537; color:#ffffff; }
+
+.sidebar_layout_three .cesis_widget_title:after { background:#efa537; }
+
+
+
+/*--------------------------------------------------------------
+#7 Page Settings
+--------------------------------------------------------------*/
+
+/* width settings */
+.page .site-main .cesis_container,.page .cesis_top_banner .cesis_container { max-width:1250px; }
+
+/* top and bottom padding settings */
+
+
+.page .article_ctn,.page .sidebar_ctn { padding-top:0; padding-bottom:0; }
+
+
+/*--------------------------------------------------------------
+#8 Blog Settings
+--------------------------------------------------------------*/
+
+
+
+/* width settings */
+.single-post .site-main .cesis_container,
+.single-post .cesis_top_banner .cesis_container{ max-width:1250px; }
+
+/* top and bottom padding settings */
+.single-post .article_ctn,.single-post .sidebar_ctn { padding-top:60px; padding-bottom:60px; }
+
+/* stacked gallery */
+
+.single-post .cesis_blog_gallery_stacked .cesis_gallery_img img { margin-bottom:0px;}
+
+/* width settings */
+body.blog .site-main .cesis_container,.archive.category .site-main .cesis_container,.archive.tag .site-main .cesis_container,
+body.blog .cesis_top_banner .cesis_container,.archive.category .cesis_top_banner .cesis_container,.archive.tag .cesis_top_banner .cesis_container { max-width:1250px; }
+
+/* top and bottom padding settings */
+body.blog .article_ctn,body.blog .sidebar_ctn,.archive.category .article_ctn,.archive.category .sidebar_ctn,.archive.tag .article_ctn,.archive.tag .sidebar_ctn,.archive.date .article_ctn,.archive.date .sidebar_ctn{ padding-top:60px; padding-bottom:60px; }
+
+
+
+
+
+/*--------------------------------------------------------------
+#9 Portfolio Settings
+--------------------------------------------------------------*/
+
+
+
+
+/* width settings */
+
+/* top and bottom padding settings */
+
+
+
+/* stacked gallery */
+
+
+/* width settings */
+
+/* top and bottom padding settings */
+
+
+
+/*--------------------------------------------------------------
+#10 Staff Settings
+--------------------------------------------------------------*/
+
+
+
+
+/* width settings */
+
+/* top and bottom padding settings */
+
+
+
+/* width settings */
+
+/* top and bottom padding settings */
+
+
+
+
+
+/*--------------------------------------------------------------
+#11 Career Position Settings
+--------------------------------------------------------------*/
```

Comment:
Let's get rid of all empty blocks with empty section comment delimiters, here and elsewhere.